tiup: add close swap section and fix a command (#2505)
pingcap/docs: how-to/deploy/orchestrated/tiup.md

    ```bash
    tiup cluster exec tidb-test --sudo --command "yum -y install numactl"
    ```

### How to disable system swap

TiDB requires sufficient memory space for operation. It is not recommended to use swap as a buffer for insufficient memory, which might reduce performance. Therefore, it is recommended to disable the system swap permanently.

> Note that if you disable the system swap by using `swapoff -a`, the setting will be invalid after the machine is restarted.

Reviewer: ```suggestion
> **Note:**
>
> Do not disable the system swap by executing `swapoff -a`, or this setting will be invalid after the machine is restarted.
```
